content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -143,10 +143,6 @@ network complexity required to deploy Kubernetes at scale within AWS.
 [Coil](https://github.com/cybozu-go/coil) is a CNI plugin designed for ease of integration, providing flexible egress networking.
 Coil operates with a low overhead compared to bare metal, and allows you to define arbitrary egress NAT gateways for external networks.
 
-### Contiv
-
-[Contiv](https://github.com/contiv/netplugin) provides configurable networking (native l3 using BGP, overlay using vxlan,  classic l2, or Cisco-SDN/ACI) for various use cases.
```

Comment:
Should we remove this because there have been no commits for 2 years? Typically, we try not to pass judgement on projects. We don't require network plugins to have a shiny website, either.

If there were evidence of a malicious takeover, or the software didn't work, that might be stronger grounds for removal.

Reply:
I am not in favor of removing entries based on their repo not have a commit in two years, but 4+ years seems a bit long to go without any update at all.  When you add that to the fact that the domain name doesn't point to the right content (it rotates between a marketing squatter and forcing you to install a browser plugin via multiple redirects, it makes it more clear that this is abandonware.

Also, this project was for demo purposes only and displayed at Cisco Live 2017.  It's not even mentioned on the official website.

Reply:
@sftim Can we move forward with this?